notification/build.gradle

```diff
@@ -15,6 +15,8 @@
 
 apply plugin: 'java'
 apply plugin: 'jacoco'
+apply plugin: 'maven-publish'
```

Comment:
basing off this [Job-Scheduler PR](https://github.com/opendistro-for-elasticsearch/job-scheduler/commit/538a5c2e6c642ac5bb8ec6059953c7df862b6846) this change doesn't have `signing`. Do we need to also add this?